xbmc/cores/VideoPlayer/DVDCodecs/Video/VAAPI.cpp

```diff
@@ -123,14 +128,44 @@ bool CVAAPIContext::EnsureContext(CVAAPIContext **ctx, CDecoder *decoder)
   return true;
 }
 
+namespace
+{
+int find_open_render_node_fd()
```

Comment:
Reading this:
https://dvdhrm.wordpress.com/tag/drm/

It states a few times:
"It’s also important to know that render-nodes are not bound to a specific card."
"Instead, if user-space requires hardware-acceleration, it should open any node and use it."
"Really! Questions like “how do I find the render-node for a given card?” don’t make any sense. Yes, driver-specific user-space can figure out whether and which render-node was created by which driver, but driver-unspecific user-space should never do that!"

From that, and since kodi should be driver unspecific this should be a perfect use case of the any render node?